integration/scripts/deploy.sh

```diff
@@ -45,17 +45,17 @@ deploy_dev() {
 deploy_xpu() {
     echo "Deploying xPU environment"
     echo "BMC IP address: ${BMC_IP_ADDRESS}"
-    echo "Host IP address: ${INFRA_IP_ADDRESS}"
+    echo "Infra IP address: ${INFRA_IP_ADDRESS}"
     echo "xPU IP address: ${XPU_IP_ADDRESS}"
 
-    # Deploy to host/VM/container
-    export COMPOSE_FILE=docker-compose.pxe.yml:docker-compose.spdk.yml
-    export DOCKER_HOST="ssh://user@${INFRA_IP_ADDRESS}"
+    # Deploy to laptop/VM/server
+    export COMPOSE_FILE=docker-compose.otel.yml:docker-compose.spdk.yml:docker-compose.pxe.yml:docker-compose.networks.yml
+    [[ -n ${INFRA_IP_ADDRESS} ]] && export DOCKER_HOST="ssh://ubuntu@${INFRA_IP_ADDRESS}"
     bash -c "${DC} up -d"
 
     # Deploy to XPU
-    export COMPOSE_FILE=docker-compose.xpu.yml
-    export DOCKER_HOST="ssh://user@${XPU_IP_ADDRESS}"
+    export COMPOSE_FILE=docker-compose.xpu.yml:docker-compose.networks.yml
+    export DOCKER_HOST="ssh://ubuntu@${XPU_IP_ADDRESS}"
```

Comment:
Same comment on `ubuntu` here. We need this to be configurable. Mergable now, but we can fix post-merge.

Reply:
yep